webblog/frontend/src/components/blog/Dashboard.js

```diff
@@ -0,0 +1,12 @@
+import React, { Fragment } from "react";
+import Form from "./Form";
+import BlogList from "./BlogList";
+
+export default function Dashboard() {
+  return (
+    <Fragment>
```

Comment:
What is the reason to use Fragment?

Reply:
Fragment is used to combine multiple children into a single tag without adding every single component to be added separately to the DOM.

ref: https://reactjs.org/docs/fragments.html